### ISSUE TITLE
Skins Web Tool - POC / WIP

### ISSUE THREAD

**Issue by Ombridride** [Ombridride/minetest-minetestforfun-server#52]
Very difficult for now to speak about this future tool...
(Login Screen)
![Login](http://i.imgur.com/PMRnEJq.png)
(Skin Changer/Uploader Screen)
![Skin Changer/Uploader Screen](http://i.imgur.com/ho8QpTZ.png)

@davedevils has posted his work on the forum : https://forum.minetest.net/viewtopic.php?f=14&t=11873

but we can detail the project and say the missing things :

- [x] Make a Proof Of Concept version of the tool
- [ ] Make a Work In Progress version of the tool
- [ ] Add this tool to the website (sub-domain)
- [ ] Traduct the tool in english
- [ ] Make the first Release version of the tool

**Comment by ObaniGemini** [comment]
How do you do the render of the image? Do you use a background and put a skin on it?

**Comment by ObaniGemini** [comment]
I can make an English traduction, but where are the sources?

**Comment by Ombridride** [comment]
Don't waste your time with the traduction, it's very fast and easy, i will do it when i will work on this thing ;)

**Comment by ObaniGemini** [comment]
ok

**Comment by Ombridride** [comment]
@davedevils 
No more news, is the project still in development ? :)

**Comment by Ombridride** [comment]
Issue assigned to gravgun, he wants to continu/redo the Dave work ;)

**Comment by gravgun** [comment]
I'll probably redo the whole thing (maybe to the exception of stylesheets), the current foundation is very small and was visibly coded in a short time. Another main reason is that I could use this project to show off my PHP skills and, as of now, ability to create a fairly basic but solid web interface.

**Comment by gravgun** [comment]
Progress update: I'm quite slow creating the panel actually, but I've got some things worked out such as login using actual MT server accounts.
![PPanel preview](http://i.imgur.com/0pZnOOv.png)

**Comment by davedevils** [comment]
Back to MineTest , I can finish the webite if you want (exept if gravgun no want )

I have redo a new version 

More explain here :
https://forum.minetest.net/viewtopic.php?f=14&t=11873&p=195518#p195518

**Comment by Ombridride** [comment]
Hi @davedevils, happy to see you again !
@gravgun did some great things and improvements, maybe you can work together in this panel ?

**Comment by Ombridride** [comment]
@gravgun Please, Can you create a repository inside the "MinetestForFun team", it will grant us the possibility to follow easily your work / help you / add opinions / issues / etc...

**Comment by gravgun** [comment]
@Ombridride https://github.com/MinetestForFun/ppanel

**Comment by Ombridride** [comment]
@ElementW Do you planned to finish this work ?

**Comment by ElementW** [comment]
@Ombridride Yes! And this is probably the only thing left in relation with MT I'll finish and maintain. But to do it I need to talk to you to set it up to see if anything breaks in this production(-ish) environment. Don't worry, this is standalone and only writes to its sqlite DB, everything else is read only, so it won't break anything.

**Comment by Darcidride** [comment]
@ElementW 
I've a solution in mind, no need to put parts of your POC on the webserver and on minetest, everything on Minetest should work, can we peak about the finallity of this project soon ? :)

**Comment by Darcidride** [comment]
@ElementW I don't think we already talk about this tool, well, it's a reminder :)

**Comment by davedevils** [comment]
No news about that ?
I can rework on the project , if @ElementW  stop to make it.

**Comment by ObaniGemini** [comment]
You should come on IRC then :)
@ElementW seems very busy those times and doesn't come on IRC much

**Comment by Darcidride** [comment]
@davedevils Can when you want on our IRC, we will speak about it.
But yes, currently this feature isn't done
